movies/ui_movies.py

```diff
@@ -1,5 +1,51 @@
+import time
+import ui.main as ui
+import csv
+
 def call_add_movie():
-    pass
+    ui.clear()
+    print('Add Movie')
+    print()
+    movie_title = input("Movie Title: ")
+    movie_release_date = input("Movie Release Date: ")
+    rating = input("Rating: ")
+    try:
+        number_of_units = int( input ("How many do you have in stock?: "))
+    except:
+        print(ui.choice_invalid_messaging)
+        time.sleep(ui.invalid_timer)
+
+    fields = ['movie_title', 'movie_release_date', 'rating', 'number_of_units']
+    rows = [[movie_title, movie_release_date, rating, number_of_units]]
+    
+    with open('data/movie.csv') as csv_file:
```

Comment:
@simpletech310 Found that if the movie.csv file does not exist, the system will crash.